src/docs/product/integrations/issue-tracking/jira/index.mdx

```diff
@@ -158,6 +158,24 @@ If you hit a 4xx or 5xx error during or after setting up the Jira Server integra
 
 Alert notifications in Sentry can be routed to many supported integrations, but by default are aimed at email. You will need to configure a project’s [**Alert Rules**](/product/alerts-notifications/alerts/) to properly route notifications to a specific integration.
 
+### Ignored Fields
+
+Ignored fields are Jira fields that are hidden in the issue creation form. To add ignored fields, navigate to **Organization Settings > Integrations** and click "Configure" next to your Jira instance. Then scroll down to “Ignored Fields”.
+
+Type the id or name of the Jira field, not the label, into the text input box. The following section will state how to find the field id or name.
+
+#### Find the id of the Field
+
+1. In Jira, access a ticket that has the field that you want the id or name of
+2. Right click the screen and select “Inspect” or click F12
+3. Click the “Elements” tab of the Inspect module
+4. Click the “Select an element” button on the top left of the Inspect module
+   ![Select an element button](select-element.png)
+5. Click the field that you want the id or name of
+6. View the name or id of the field
+   a. The name or id of the field is the last part of the ‘data-testid’ field
+      ![Jira field id or name](jira-field-id-name.png)
```

Comment:
Just FYI, you can just number these all as 1. and when it builds, it will increment the numbers correctly.

```suggestion
1. In Jira, access a ticket that has the field that you want the ID/name of.
2. Right click the screen and select “Inspect” or click F12.
3. Click the “Elements” tab of the "Inspect" module.
4. Click the “Select an element” button on the top left of the "Inspect" module.
   ![Select an element button](select-element.png)
5. Click the field that you want the ID/name of.
6. Find the last part of the ‘data-testid’ field, which is the ID/name of the field.
      ![Jira field id or name](jira-field-id-name.png)
```